sycl/test-e2e/KernelCompiler/kernel_compiler_spirv.cpp

```diff
@@ -10,6 +10,7 @@
 
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out %S/Kernels/kernels.spv %S/Kernels/kernels_fp16.spv %S/Kernels/kernels_fp64.spv
+// XFAIL: cpu
```

Comment:
Likewise.

Reply:
open https://github.com/intel/llvm/issues/13397 for the new marked 'XFAIL' tests.